packages/front-end/components/Experiment/NewExperimentForm.tsx

```diff
@@ -227,33 +226,28 @@ const NewExperimentForm: FC<NewExperimentFormProps> = ({
         ? initialValue.variations
         : getDefaultVariations(initialNumVariations),
       phases: [
-        initialValue
-          ? {
-              coverage: initialValue.phases?.[0].coverage || 1,
-              dateStarted: getValidDate(
-                initialValue.phases?.[0]?.dateStarted ?? ""
-              )
-                .toISOString()
-                .substr(0, 16),
-              dateEnded: getValidDate(initialValue.phases?.[0]?.dateEnded ?? "")
-                .toISOString()
-                .substr(0, 16),
-              name: initialValue.phases?.[0].name || "Main",
```

Comment:
I think this other code here is important to keep for importing past experiments.  I'm pretty sure the `getValidDate` stuff at the very least was added because of a bug.